controller/test/ControllerRTL_test.py

```diff
@@ -0,0 +1,127 @@
+'''
+=========================================================================
+ChannelRTL_test.py
+=========================================================================
+Simple test for Channel.
+
+Author : Cheng Tan
+  Date : Dec 2, 2024
+'''
+
+
+from pymtl3 import *
+from pymtl3.stdlib.test_utils import TestVectorSimulator
+from ..ControllerRTL import ControllerRTL
+from ...lib.basic.en_rdy.test_sinks import TestSinkRTL
+from ...lib.basic.en_rdy.test_srcs import TestSrcRTL
+from ...lib.basic.val_rdy.SourceRTL import SourceRTL as TestValRdySrcRTL
+from ...lib.basic.val_rdy.ifcs import SendIfcRTL as ValRdySendIfcRTL
+from ...lib.basic.val_rdy.ifcs import RecvIfcRTL as ValRdyRecvIfcRTL
+from ...lib.messages import *
+from ...noc.PyOCN.pymtl3_net.ocnlib.test.stream_sinks import NetSinkRTL as TestNetSinkRTL
+import pytest
+
+
+#-------------------------------------------------------------------------
+# TestHarness
+#-------------------------------------------------------------------------
+
+class TestHarness(Component):
+
+  def construct(s, MsgType, PktType, AddrType, src_msgs, sink_msgs, src_pkts, sink_pkts):
+
+    cmp_fn = lambda a, b : a.payload == b.payload
+    s.src_en_rdy = TestSrcRTL(MsgType, src_msgs)
+    s.sink_en_rdy = TestSinkRTL(MsgType, sink_msgs)
+    s.src_val_rdy = TestValRdySrcRTL(PktType, src_pkts)
+    s.sink_val_rdy = TestNetSinkRTL(PktType, sink_pkts, cmp_fn = cmp_fn)
+
+    s.dut = ControllerRTL(PktType, MsgType, AddrType)
+
+    # Connections
+    s.src_en_rdy.send //= s.dut.recv_from_master
```

Comment:
Should it be `s.dut.recv_from_master //= s.src_en_rdy.send`,  input signal on the left?

Reply:
From what I understand, when we use `//=`, it means signals on both sides would be driven together (or drive other together). It is `connection` instead of `assignment`.